docs/services/navigation/_index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "The Navigation Service"
+linkTitle: "Navigation"
+description: "The Navigation Service uses GPS to autonomously navigate a rover to user defined endpoints."
+type: docs
+weight: 45
+no_list: true
+# icon: "/services/img/icons/frame-system.svg"
+tags: ["navigation", "services", "base", "rover"]
+# SMEs: Raymond
+---
+
+The Navigation service is the stateful definition of Viam's [Motion service](/services/motion/)
+It uses GPS to autonomously navigate a rover [base](/components/base/) to user defined endpoints called `Waypoints`.
+Once the `Waypoints` are added and the `Mode` of the service is [set to `waypoints`](#setmode), Navigation services calls begin.
```

Comment:
what does Navigation services calls begin mean - does that mean "the rover/base begins to navicate to the waypoint(s)"?

Reply:
I think that is what it means but that explanation confused me as well. I think this might be better but it's a bit long, I could trim the top info if you think that could help?